entrypoint.sh

```diff
@@ -86,31 +90,37 @@ else
   COMMIT_DESC="Includes transpiled JS/TS."
 fi
 
-git add dist/* -f
+# Only commit if we choose to do so
+# Useful for validating that JS is syntactically valid
+if [[ "$INPUT_DO_NOT_COMMIT" == "false" ]]; then
```

Comment:
nit: Can we switch the order so the "do not commit" case comes first for readability?